ci/gpu/build.sh

```diff
@@ -56,7 +50,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=${MINOR_VERSION}" \
-      "xgboost=${XGBOOST_VERSION}" \
+      "xgboost=1.2.0dev.rapidsai0.15" \
```

Comment:
should this be version `0.16` since the PR is merging into `branch-0.16`? I

Reply:
Yeah it's probably due to the fact this was based off `branch-0.15`. This should go away once it is rebased on `branch-0.16`.